src/FSharp.Control.TaskSeq/TaskSeq.fs

```diff
@@ -162,6 +162,12 @@ type TaskSeq private () =
     // Utility functions
     //
 
+    static member max source = Internal.maxMin max source
+    static member min source = Internal.maxMin max source
```

Comment:
```suggestion
    static member min source = Internal.maxMin min source
```

Reply:
🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣

Reply:
That's what you get from doing things at 2AM 😆

Reply:
Fixed

Reply:
You'd be paying more attention if you didnt intend to write tests
And you were clearly following the dictum of seeing the test fail first.